exercises/5.3/solution_ws/ros2/src/py_perception/src/py_perception_node.cpp

```diff
@@ -0,0 +1,353 @@
+#include "py_perception/srv/filter_cloud.hpp"
```

Comment:
Maybe instead of providing a duplicate of the 5.1 template workspace (that needs to be kept up to date), we should have them copy from the 5.1 template / solution?